opennms-webapp/src/main/java/org/opennms/netmgt/config/attrsummary/descriptors/AttributeDescriptor.java

```diff
@@ -0,0 +1,327 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ * 
+ * Copyright (C) 2017-2017 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2017 The OpenNMS Group, Inc.
+ * 
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ * 
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *     http://www.gnu.org/licenses/
+ * 
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.netmgt.config.attrsummary.descriptors;
+
+
+import org.opennms.netmgt.config.attrsummary.Attribute;
+
+/**
+ * Class AttributeDescriptor.
+ * 
+ * @version $Revision$ $Date$
+ */
+@SuppressWarnings("all") public class AttributeDescriptor extends org.exolab.castor.xml.util.XMLClassDescriptorImpl {
```

Comment:
Can we remove the descriptors too? We want to be able to remove all the Castor references.